cmd/ui/src/components/GraphEvents.tsx

```diff
@@ -301,8 +311,18 @@ export const GraphEvents = forwardRef(function GraphEvents(
     }, [draggedNode, setSettings, showEdgeLabels]);
 
     useEffect(() => {
-        resetCamera(sigma);
-    }, [sigma]);
+        if (sigmaChartRef?.current) {
+            if (exploreLayout === 'sequential') {
+                sigmaChartRef?.current?.runSequentialLayout();
+            } else if (exploreLayout === 'standard') {
+                sigmaChartRef?.current?.runStandardLayout();
+            } else {
+                resetCamera(sigma);
+            }
+        } else {
+            resetCamera(sigma);
+        }
+    }, [sigma, exploreLayout, sigmaChartRef]);
```

Comment:
Is this to handle the initial layout when the page is first rendered? If so, in the `initGraph` util called in GraphView we managed the initial default layout (`cmd/ui/src/views/Explore/utils.ts` line 85) previously and i would think we should consolidate the logic to either move this check to initGraph, or remove the check in initGraph?

Reply:
discussed and resolved outside of PR!